app/experimenter/experiments/tasks.py

```diff
@@ -210,6 +210,23 @@ def send_period_ending_emails(experiment):
                 "Sent ending email for Experiment: {}".format(experiment)
             )
 
+    # send enrollment ending emails if enrollment end
+    # date is 5 days out
+    if experiment.enrollment_end_date:
+        if (experiment.enrollment_end_date - date.today()) <= timedelta(
```

Comment:
These two checks, the date check and the email exists check, both only need the experiment instance and some constants, so in theory it could be an experiment instance method?

Reply:
Yes, I see what you mean about that...